crates/turborepo-lib/src/daemon/connector.rs

```diff
@@ -0,0 +1,278 @@
+use std::{
+    path::{Path, PathBuf},
+    process::Stdio,
+    sync::Arc,
+    time::Duration,
+};
+
+use command_group::AsyncCommandGroup;
+use log::{debug, error};
+use notify::{Config, Event, EventKind, RecommendedWatcher, Watcher};
+use sysinfo::{ProcessExt, ProcessRefreshKind, RefreshKind, SystemExt};
+use tokio::{net::UnixStream, sync::mpsc, time::timeout};
+use tonic::transport::Endpoint;
+
+use super::{client::proto::turbod_client::TurbodClient, DaemonClient, DaemonError};
+
+#[derive(Debug)]
+pub struct DaemonConnector {
+    pub dont_start: bool,
```

Comment:
I find it a little confusing having negatives as booleans. Could we rephrase these as positives?

Reply:
I'm like 90% sure this is because the Go implementation cares about zero-states.

Reply:
Agreed. Decided to keep the old phrasing but I will change it :)